README.md

```diff
@@ -61,3 +61,8 @@ To deploy the project to the live internet, you can use
 For a more in-depth getting started guide, visit the
 [Getting Started](https://fresh.deno.dev/docs/getting-started) page in the Fresh
 docs.
+
+## Badges
+
+![Made with Fresh](./www/static/fresh-badge.svg)
+![Made with Fresh(dark)](./www/static/fresh-badge-dark.svg)
```

Comment:
Some ready-to-copy markdown and HTML / JSX code snippets for embeds that link back to https://fresh.deno.dev would be great. Ie

```html
<a href="https://fresh.deno.dev"><img width="197" height="37" src="https://fresh.deno.dev/fresh-badge.svg" /></a>
```

Reply:
Yeah! it would be great ux!